src/components/core/Drawer.vue

```diff
@@ -3,32 +3,26 @@
     id="app-drawer"
     v-model="inputValue"
     app
-    dark
+    light
     floating
     persistent
     mobile-break-point="991"
     width="260"
+    clipped
```

Comment:
I did see a change with & without ``clipped`` - it seems we are seeing different behaviour, perhaps due to different browser choice? I am using Firefox (60.7.2esr, 64-bit, if that is relevant).

Without ``clipped``, the Cylc logo was always cut off by the top bar of my browser, even when I tried changing most of the logo & image attributes & ``margin-...`` specs in an attempt to prevent this is in a more intuitive way. Only setting ``clipped`` seemed to work.

Reply:
Ah! Interesting! Thanks for the explanation Sadie!